README.md

```diff
@@ -137,10 +137,62 @@ ipc-benchmark --percentiles 50 90 95 99 99.9 99.99
 # TCP-specific configuration
 ipc-benchmark -m tcp --host 127.0.0.1 --port 9090
 
+# POSIX Message Queue-specific configuration
+ipc-benchmark -m pmq --pmq-priority 1
+
 # Shared memory configuration (demonstrating a user-provided buffer size)
 ipc-benchmark -m shm --buffer-size 16384
 ```
 
+### First-Message Latency (Canary)
+
+The first message in any benchmark often has a higher latency than subsequent messages due to "cold start" effects like CPU cache misses, memory allocation, and branch prediction misses. To provide more stable and representative results, this tool automatically sends one "canary" message before starting the measurement loop. This message and its latency are discarded by default.
+
+If you need to analyze the raw performance data, including the first-message spike, you can use the `--include-first-message` flag to disable this behavior.
+
+```bash
+# Include the first message in the final results
+ipc-benchmark --include-first-message
```

Comment:
I wonder if there is any value in including the statistics of first message separately, regardless of if it is included or not.

Reply:
I spawned this off into a new issue for review.